channels-rke2.yaml

```diff
@@ -1,11 +1,11 @@
 releases:
-  - version: v1.19.14+rke2r1
+  - version: v1.19.15+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.20.10+rke2r1
+  - version: v1.20.11+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.21.4+rke2r2
+  - version: v1.21.5+rke2r2
```

Comment:
This is not how you update a fully released RKE2.

Reply:
Apologies on the that, fix.